### ISSUE TITLE
documentation empty

### ISSUE THREAD

**Issue by ConnyOnny** [owncloud/pyocclient#149]
The generated html documentation is empty (i.e. the generated html files contain no information whatsoever on the usage of pyocclient).

What I did:

```
git clone https://github.com/owncloud/pyocclient.git
pip2 install --user sphinx
cd docs
make html
firefox build/html/index.html
```

All these commands reported to be working correctly without any complaints.

Attached you find a screenshot of the generated "owncloud.html" file, which I suppose should contain a bunch of documentation.

![screenshot - 26 02 2016 - 13 53 45](https://cloud.githubusercontent.com/assets/1289200/13352861/9346013a-dc90-11e5-813e-066472f1dc6d.png)

**Comment by PVince81** [comment]
Indeed. Weird.

**Comment by PVince81** [comment]
Fixed: 358339258bb4d15c0b91f82c29232310a1985b11
